client/lib/stats/stats-list/test/index.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External Dependencies
+ */
+import { assert } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal Dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'StatList', () => {
+	let StatList, statsParser, data
+
+	useMockery( mockery => {
+		mockery.registerMock( 'lib/wp', require( './mocks/lib/wp' ) );
+		mockery.registerMock( 'lib/user', require( './mocks/lib/user' ) );
+	} );
+
+	useFakeDom();
```

Comment:
This one does have a dom dependent requirement , `node_modules/store/store.js`. I can mock that module or keep the fake dom. Not sure what the performance hit would be in the former since I think mockery can slow things down a bit.